### ISSUE TITLE
Request-URI Too Long on Edit categories on exports profile

### ISSUE THREAD

**Issue by nei** [akeneo/pim-community-dev#6954]
We are having exactly the same issue as this one https://github.com/akeneo/pim-community-dev/issues/6153 but  on the exports page, the modal where you select the categories does not work when you try to edit a profile that has too many checkboxes selected.

On this example we have 240 categories selected where the url is 17k chars.

```
https://akeneo.dev/enrich/category/rest/website/selected-children?selected%5B%5D=toolsworkshop&selected%5B%5D=x_tools_still_to_be_categorised&selected%5B%5D=toolsworkshopweldingcutting&selected%5B%5D=toolsworkshoptoolboxeschestsstorage&selected%5B%5D=toolsworkshopsafetyworkwear&selected%5B%5D=toolsworkshoppowertoolsaccessories&selected%5B%5D=toolsworkshopliftingequipment&selected%5B%5D=toolsworkshophomegardenleisure&selected%5B%5D=toolsworkshophandtools&selected%5B%5D=toolsworkshopgarageequipment&selected%5B%5D=toolsworkshopclearance&selected%5B%5D=toolsworkshopairtoolscompressors&selected%5B%5D=toolsworkshopweldingcuttingweldingprotectionsafety&selected%5B%5D=toolsworks
```
I believe we should change this for POST instead of GET too.

**Comment by LaureBro** [comment]
Hi Nei, 
thanks for reporting this, but what's your PIM version?
Regards, 
Laure

**Comment by nei** [comment]
Sorry. We are using 1.7.1

**Comment by LaureBro** [comment]
Hello @nei, @muspelheim, sorry for our late feedback! This issue has been fixed by PR #8278 and #8382 in 2.0, 2.2 and 2.3 versions.
I'm closing this issue, feel free to reopen it if you still experience this issue.

Regards,
Laure
